doc/index.rst

```diff
@@ -1,8 +1,45 @@
-Plutus
-======
+Plutus Programming and Reference Guide
+=============================================
+
+About Plutus
+--------------------
+
+The Plutus project consists of Plutus Core, the programming language used for scripts on Cardano; tooling and compilers for compiling various intermediate languages into Plutus Core;
+and Plutus Tx, the compiler that compiles the Haskell source code into Plutus Core 
+to form the on-chain part of a contract application. All of this is used in combination 
+to write Plutus Core scripts that run on the Cardano blockchain. Compiled Plutus Core 
+scripts are able to interact with the Cardano ledger through the 
+ledger interface. 
+
+The Plutus Core Repository
+----------------------------------
+
+The `Plutus Core Repository <https://github.com/input-output-hk/plutus>`_ contains 
+the implementation, specification, and mechanized metatheory of Plutus Core. 
+It also contains Plutus Tx. 
+
+About This Documentation
+---------------------------------
+
+The purpose of this documentation is to serve as an introduction to the Plutus language 
+and to programming using Plutus Tx. It is meant to be educational, so it includes explanations, 
+tutorials and how-to instructions. It is also intended as a reference for beginning and 
+experienced developers. 
+
+Intended Audience
+---------------------------
+
+The intended audience of this documentation includes DApp developers, Plutus developers, 
+developers writing smart contracts, Haskell developers who want to write Plutus scripts, 
+and developers who want to learn to use the compiler, Plutus Tx. It is also designed for 
+developers who want to learn Plutus directly from the team that developed the Plutus language. 
+
+Because Plutus Tx is built on top of Haskell, it is helpful to have a background in Haskell. 
```

Comment:
I am not sure what it means to say plutus TX is builtin on top of haskell. Personally I try to avoid getting into whether plutus is a language or not and say instead that plutus contracts are written in Haskell.

Reply:
@jmchapman This page (https://plutus.readthedocs.io/en/latest/tutorials/plutus-tx.html) gets into a bit of detail on this topic. My impression at this point is that we don't need to shy away from this type of statement. Although how much attention we should bring to it is another matter. 

I think it would be very helpful to come up with a list of top-level statements that are the most useful and helpful to people. For example, saying that Plutus contracts are written in Haskell is a pretty straightforward statement and easy to understand. 

I also think it's important to be technically accurate, so long as it doesn't become an obstacle to our users. Maybe levels of technical accuracy balanced with levels of helpful communication would be worth thinking about.